medmodels/medrecord/querying.pyi

```diff
@@ -175,19 +179,62 @@ class SingleValueOperand:
     ) -> None: ...
     def clone(self) -> SingleValueOperand: ...
 
+class AttributesTreeOperand:
+    def max(self) -> MultipleAttributesOperand: ...
+    def min(self) -> MultipleAttributesOperand: ...
+    def count(self) -> MultipleAttributesOperand: ...
+    def sum(self) -> MultipleAttributesOperand: ...
+    def first(self) -> MultipleAttributesOperand: ...
+    def last(self) -> MultipleAttributesOperand: ...
+    def is_string(self) -> None: ...
+    def is_int(self) -> None: ...
```

Comment:
why no is_float, is_datetime, is_null or is_bool here?

Reply:
A `MedRecordAttribute` can only be a `string` or `int`